### DISCUSSION TITLE
Is it possible to make MIDI controls?

### DISCUSSION THREAD

**Discussion by vincehi** [rxhanson/Multitouch-Community#116] [General]
Is it possible to make MIDI controls? And if yes, is it possible to divide in 8 for example to recreate a MIDI pad?
thk's

**Comment by rxhanson** [comment]
There's nothing in the app built specifically for MIDI at this time. I've thought about this type of implementation before, though. Would you be putting some kind of guidelines on your trackpad so you know which input you're on?

**Comment by vincehi** [comment]
Why not but not necessarily.

https://www.youtube.com/watch?v=JKz9pAcLEUk
